pkg/keydb/keydb.go

```diff
@@ -2,13 +2,20 @@ package keydb
 
 import (
 	"github.com/boltdb/bolt"
+	"github.com/cashweb/keyserver/pkg/models"
 )
 
 type KeyDB struct {
 	db *bolt.DB
 }
 
+type Database interface {
```

Comment:
I thought about doing this, but the recommendation is to "Accept an interface, return a struct" Which kinda makes sense.